dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -9720,12 +9720,13 @@ components:
         type:
           type: string
           enum:
-          - SMK
```

Comment:
Question: isn't this an auto-generated on build file? I'm a little confused how this occurred without a corresponding Java code change.